src/huggingface_hub/hf_api.py

```diff
@@ -7566,9 +7566,9 @@ def create_inference_endpoint(
         region: str,
         vendor: str,
         account_id: Optional[str] = None,
-        min_replica: int = 0,
+        min_replica: int = 1,
         max_replica: int = 1,
-        scale_to_zero_timeout: int = 15,
+        scale_to_zero_timeout: int | None = None,
```

Comment:
the PEP 604 syntax won't work in python 3.8 (we still support python 3.8 in hfh).
```suggestion
        scale_to_zero_timeout: Optional[int] = None,
```

Reply:
Ah, I figured you used `from __future__ import annotations`! Will update